README.md

```diff
@@ -28,11 +28,17 @@ The engine can be configured through several variables, each of which has a pres
 - `TESTSCRIPT_PATH`: The relative path to the directory containing the TestScript resources (as JSON or XML) to be executed by the engine. If any TestScript in the directory uses a fixture, the directory MUST also include a `fixtures` subfolder containing files whose relative paths match the reference value of a fixture within a TestScript.
 - `TESTREPORT_PATH`: The relative to the directory containing the TestReports output following their partner TestScript execution.
 
-### Details
+### Commandline Arguments
+
+Some configurations can be set using command line arguments:
+- `-r` or `--runnable`: the next argument must be the name of runnable to execute. Only works if the noninteractive flag is provided.
+- `-n` or `--noninteractive`: disable confirmation of configuration settings
```

Comment:
I would add in an example of using both of these. something like `bundle exec bin/testscript_engine -n -r "TestScript Example History"`. Took me a couple tries to figure out the proper way to include the name of the runnable.